src/NCronJob/Configuration/DynamicJobFactory.cs

```diff
@@ -36,7 +36,8 @@ public DynamicJobFactory(IServiceProvider serviceProvider, Delegate jobAction)
         var param = Expression.Parameter(typeof(object[]), "args");
         var args = method.GetParameters().Select((p, index) =>
             Expression.Convert(Expression.ArrayIndex(param, Expression.Constant(index)), p.ParameterType)).ToArray();
-        var call = Expression.Call(Expression.Constant(jobDelegate.Target), method, args);
+        var instance = method.IsStatic ? null : Expression.Constant(jobDelegate.Target);
```

Comment:
Good catch :D

Reply:
Can you add a test for this?
As far as I understand this should be trivial, just add a `static` in front:
```csharp
instantJobRegistry.RunInstantJob(static async (ChannelWriter<object> writer ) => { ... });
```

Reply:
Done